### PR TITLE
Create RoleToAccessProfiles-nonprod.json

### PR DESCRIPTION
**Pull request by MMNycz**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-3827

### Change description ###

I Add RoleToAccessProfile-noprod file WA R3 Map access profiles for specific and challenged access user roles

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,12 @@
   "scripts": {
     "lint": "eslint e2e",
     "test": "npx codeceptjs run --steps",
-    "test:smoke": "MOCHAWESOME_REPORTFILENAME=smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml npx codeceptjs run --grep @smoke-tests --reporter mocha-multi --verbose",
```
Comment (karthick-mohanasundaram-HMCTS) [inline package.json:8]: could you please revert this changes?

File: bin/add-roles.sh [hunk 2]
```diff
@@ -7,6 +7,9 @@
 ./bin/utils/ccd-add-role.sh "prd-admin"
 ./bin/utils/ccd-add-role.sh "judge-profile"
 ./bin/utils/ccd-add-role.sh "basic-access"
+./bin/utils/ccd-add-role.sh "admin-access"
+./bin/utils/ccd-add-role.sh "GS_profile"
+./bin/utils/ccd-add-role.sh "legal-adviser"
```
Comment (karthick-mohanasundaram-HMCTS) [inline bin/add-roles.sh:12]: Could you please double check whether it is legal-adviser or legal-advisor? Thanks

Comment (MMNycz) [reply bin/add-roles.sh:12]: Hi! 
In civil-service and civil-sdk we use legal-adviser.

**Comment by karthick-mohanasundaram-HMCTS** [comment]
Could you please attach the screenshot of local test?

- [ ] Attach screenshot

File: ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,338 @@
+[
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/AuthorisationCaseState/AuthorisationCaseStateGAspec-nonprod.json:1]: Should be include Application_Closed as well to save the work in future? @smitatalawadekar-hmcts  please advise me

**Comment by MMNycz** [comment]
After changes and discard Global Search Task. 
I close that PR and create new clean one.
